.travis.yml

```diff
@@ -3,6 +3,12 @@
 
 addons:
   postgresql: "9.4"
+  apt_packages:
+    - enchant
+    - aspell
+    - aspell-en
+    - ispell
+    - iamerican
```

Comment:
@tardyp good point, thanks!

`pyenchant` can be installed without supporting `libenchant` (at least current version of `pyenchant`), so `pip install ./master[docs]` should work on any machine.

But docs building will fail with `ImportError: The 'enchant' C library was not found. Please install it via your OS package manager, or use a pre-built binary wheel from PyPI.`

I'll add something like this to `conf.py`:

```python
try:
    import enchant
except ImportError as ex:
    print("Enchant import failed: {0}\nSpell checking disabled.".format(ex), file=sys.stderr)
else:
    extensions.append('sphinxcontrib.spelling')